### PR TITLE
feat: add full class modifier

### PR DESCRIPTION
**Pull request by stefanoScalzo**
## Related Issue
Closes SAP/fundamental-styles#1734

## Description
New modifier class called `--full` to make layout grid column take the remaining space left
Due to the grid being min 1/12 it still does have a min-width of 1/12 columns
### Before:
none

### After:
Screen Shot 2020-09-30 at 9 31 07 AM

#### Please check whether the PR fulfills the following requirements

1. The output matches the design specs
- [x] All values are in `rem`
- [x] Text elements follow the truncation rules
- [x] hover state of the element follow design spec
- [x] focus state of the element follow design spec
- [x] active state of the element follow design spec
- [x] selected state of the element follow design spec
- [x] selected hover state of the element follow design spec
- [x] pressed state of the element follow design spec
- [x] Responsiveness rules - the component has modifier classes for all breakpoints
- [x] Includes Compact/Cosy/Tablet design
- [x] RTL support
2. The code follows fundamental-styles code standards and style
- [x] BEM naming convention is used
- [x] Mixins are used for repeatable code (`fd-rtl`, `fd-ellipsis`, `fd-flex`, `fd-selected`, `fd-focus`, ect.)
- [x] A11y support - keyboard support, screenreader support, proper ARIA attributes, etc.
- [x] `fd-reset()` mixin is applied to all elements
- [x] Variables are used, if some value is used more than twice.
- [x] Checked if current components can be reused, instead of having new code.
3. Testing
- [x] tested Storybook examples with "CSS Resources" `normalize` option 
- [x] tested Storybook examples with "CSS Resources" `unnormalize` option 
- [x] Verified all styles in IE11
- [x] Updated tests
4. Documentation
- [x] Storybook documentation has been created/updated
- [x] Breaking Changes [wiki](https://github.com/SAP/fundamental-styles/wiki/Breaking-Changes) has been updated in case of breaking changes.

### CODE REVIEW

File: src/layout-grid.scss [hunk 1]
```diff
@@ -73,6 +73,12 @@ $cols: 12;
   min-width: 100%;
   max-width: 100%;
 
+  &--full {
+    min-width: ( ( 1 / 12 ) *100%);
+    max-width: none;
+    flex: 1;
+  }
```
Comment (InnaAtanasova) [inline src/layout-grid.scss:80]: You have the same block of code repeated 4 times. Move it to a mixin.

**Review by InnaAtanasova — CHANGES_REQUESTED** [review]
Why is this a `feature`?

File: src/layout-grid.scss (hunk 1 above)
Comment (stefanoScalzo) [reply src/layout-grid.scss]: Fixed!

**Comment by InnaAtanasova** [comment]
@stefanoScalzo why us this a `feat`?

**Review by JKMarkowski — APPROVED** [review]
LGTM, that's the thing we were missing

**Review by InnaAtanasova — APPROVED** [review]

**Pull request merged**